karenina/visualization.py

```diff
@@ -331,13 +372,18 @@ def main():
         params['color'] = color
         curr_subject = Individual(subject_id=curr_subject_id,
                                   params=params, \
-                                  metadata={opts.treatment: df[opts.treatment]}, \
+                                  metadata={opts.treatment: row[1][3]}, \
```

Comment:
We want to avoid magic numbers like row[1][3] here. Could we identify the row header indices, bind the relevant one to a variable (treatment_idx) and get what we need from that? This can help make sure the code is readable and general.